app/ViewStores.tsx

```diff
@@ -40,19 +47,35 @@ export function ViewStores(props: {
       </div>
     );
   });
-  return (
-    <Wrapper>
-      {displayedStores}
-      <LargeButton
-        onClick={() => {
-          onCreateClick();
-        }}
-        backgroundColor={greyInactiveButton}
-      >
-        Create
-      </LargeButton>
-    </Wrapper>
-  );
+  if (storeIsSelected) {
+    return (
+      <Wrapper>
+        <UpdateStore
+          onBackClick={() => {
+            setStoreIsSelected(false);
+          }}
+          onUpdateStoreSuccess={() => {
+            setStoreIsSelected(false);
+            console.log("function to overwrite existing store data goes here");
```

Comment:
not sure if it's `ViewStores`' responsibility to know how to call the backend, I would have thought that's `UpdateStore`'s job. Don't need to change anything now though, you can just remove this in the next PR